### ISSUE TITLE
how to build a model with two input and no output label?

### ISSUE THREAD

**Issue by scut-salmon** [keras-team/keras#11066]
Hi,
I would like to build a model with two input and no output label, the example is just like thsi:
![image](https://user-images.githubusercontent.com/20534039/44964186-b77e5b00-af61-11e8-9ccd-839d7abfa98d.png)
and one of the input is image, another input is word embedding vectors, I would like to let them input two different network respectively(this two network could be simple, I was not cared about how to build it now), and output two same dimension vectors, finally I would set the loss function as mean squared error.
when I predict my image and word embedding vectors, I hope my model trained could correctly tell me the distance(MSE) of this two input
I have google it before and found the two input layer question
https://github.com/keras-team/keras/issues/148
and no output label question
https://github.com/keras-team/keras/issues/7882
It seems not really the situation I was stuck in, and I was worry that I wrote the incorrect code resulting in the wrong result.
Any suggestion or reference project would be appreciated
Thanks,
Zhang
